pkg/pkgupdates/pkgupdates.jsx

```diff
@@ -0,0 +1,532 @@
+var $ = require('jquery');
+var cockpit = require('cockpit');
+var React = require('react');
+var moment = require('moment');
+var Tooltip = require('cockpit-components-tooltip.jsx').Tooltip;
+
+const _ = cockpit.gettext;
+
+// 'available' heading is built dynamically
+const STATE_HEADINGS = {
+    'loading': _('Loading available updates, please wait...'),
```

Comment:
All translatable strings need to use double quotes. This is due to the gettext C parser. For that reason it's a good habit to make all strings use double quotes.

Either that, or you could work on a different parser for javascript. ;)